products/binaryauthorization/terraform.yaml

```diff
@@ -15,11 +15,12 @@
 overrides: !ruby/object:Overrides::ResourceOverrides
   Attestor: !ruby/object:Overrides::Terraform::ResourceOverride
     import_format: ["projects/{{project}}/attestors/{{name}}"]
+    id_format: "{{project}}/{{name}}"
```

Comment:
This is going to change the ID of the attestor resource. Is that what we want to do? I believe the default id format is {{name}}

Reply:
Good point.  I'm not really sure?  This bit of code is also referenced to generate this: https://github.com/terraform-providers/terraform-provider-google-beta/pull/1166/files#diff-2bc8338f5203dd4ca3b579979b4eb21eR160

In that chunk of code, the format string seems to be generated from this `id_format`, and the parameters passed into it seem to be coming from the self_link_url (in this case "projects/{{project}}/attestors/{{attestor}}"), which, looking at that, would make me think it should be `{{project}}/{{name}}`.

However, instead of grabbing the parameters from the self_link_url, maybe we should be taking them from `id_format`.  For consistency, that would make more sense I think, but it potentially would affect more things if we were to change it now.

Reply:
I think there are a couple different things going on here. 

The ID format of the `google_binaryauthorization_attestor` resource itself is `{{name}}`, and adding this would change that to `{{project}}/{{name}}`. It seems like that may actually be a good change, and not break anything

The ID format of the IAM resource doesn't necessarily need to match that of the resource, and probably should include all the parameters from the self_link. It makes sense to me to always include everything in the self_link as the ID so that we have all the information there. If we need a project to create one of these IAM resources, we may as well include that in the ID

Maybe it makes sense to have IAM resources construct their own id format out of the self_link parameters so it can be different than the resource ID, but I think this change is fine by itself